duckduckgo/iOS
# fix AMP related crash

Task/Issue URL: 
* https://app.asana.com/0/inbox/392891325557408/1201652903642515/1201657649513942
* https://app.asana.com/0/7643933462314/1201651234289612

Tech Design URL:
CC:

**Description**:
Fixes a crash that occurs if the AMP extractor detects and tries to reload a navigation that is not actually an AMP link. 

**Steps to test this PR**:

*Important* ensure you're using embedded Privacy Config to test this - remote one has workaround that mitigates the original crash.

1. Visit basecamp.com 
2. Click on "give basecamp a try" - should be no crash
2. Check all the amp links still work https://privacy-test-pages.glitch.me/privacy-protections/amp/



**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
